auth/store.go

```diff
@@ -114,6 +112,9 @@ type AuthStore interface {
 
 	// IsAdminPermitted checks admin permission of the user
 	IsAdminPermitted(username string) bool
+
+	// GenSimpleToken produces a simple random string
+	GenSimpleToken() (string, error)
```

Comment:
It doesn't have to belong to auth because it just generates random string. However, I think let etcdserver have it would make etcdserver a little bit dirty. But I don't have strong opinions about it. Which package is suitable for the function?

I think jwt will need another interface because it can encode username.